matomo/processors.py

```diff
@@ -0,0 +1,7 @@
+from django.conf import settings
+
+
+def matomo_tracking(request):
+    return {
+        'matomo_tracking': settings.MATOMO_TRACKING,
+    }
```

Comment:
Not getting much benefit from this processor since this context variable is only used in the tracking tag template. It would be better to move this to the templatetag.